Automattic-Tracks-iOS/CrashLogging.swift

```diff
@@ -0,0 +1,156 @@
+import Foundation
+import Sentry
+
+/// A class that provides support for logging crashes. Not compatible with Objective-C.
+public class CrashLogging {
+
+    /// A singleton is maintained, but the host application needn't be aware of its existence.
```

Comment:
I might leave that for now, just because when we start adding testing, I might need to access the `internal` initializer! 

At least we don't have to worry about consumers calling it and using it wrong :)

Reply:
👍